src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/MenuItem.tsx

```diff
@@ -1,21 +1,23 @@
 // Libraries
 import React, {FC} from 'react'
-
-// Components
 import {
   AlignItems,
   Dropdown,
   FlexBox,
   Icon,
   IconFont,
 } from '@influxdata/clockface'
+import {useDispatch} from 'react-redux'
+
+// Components
+import {dismissOverlay, showOverlay} from 'src/overlays/actions/overlays'
 
 export const CreateOrganizationMenuItem: FC = () => {
+  const dispatch = useDispatch()
   const handleCreateOrg = () => {
-    // TODO: Enable this when #5899 is worked on
-    // event('globalheader_createOrg_menuItem_clicked')
-    // TODO: Remove this comment when you are working on https://github.com/influxdata/ui/issues/5899
-    //       Add create organization functionality with modal, error handling and redirect to new organization
+    dispatch(
+      showOverlay('create-organization', null, () => dispatch(dismissOverlay()))
```

Comment:
I don't think you need the second `dispatch()` call. 
```suggestion
      dispatch(showOverlay('create-organization', null, () => (dismissOverlay()))
```

Reply:
That's true, but not for the reason we'd expect. 

The third argument to `showOverlay` is a function that is run `onClose`. However, the context provider always provides an `onClose` function that dispatches `dismissOverlay()` regardless of what we provide as the third value. I've tested it, for example, with `null` as the third argument, and there's zero change in behavior - the overlay still dismisses as expected.

TLDR, this is a bigger problem with the `showOverlay` and overlay context provider because the third argument isn't providing value. I'd like to leave `dispatch(dismissOverlay())))` in there because it at least provides some descriptive account of what is going on.